tests/components/unifiled/test_config_flow.py

```diff
@@ -0,0 +1,97 @@
+"""Test the unifiled config flow."""
+from unittest.mock import patch
+
+from homeassistant import config_entries, setup
+from homeassistant.components.unifiled.const import DOMAIN
+from homeassistant.components.unifiled.config_flow import CannotConnect, InvalidAuth
+
+from tests.common import mock_coro
+
+
+async def test_form(hass):
+    """Test we get the form."""
+    await setup.async_setup_component(hass, "persistent_notification", {})
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": config_entries.SOURCE_USER}
+    )
+    assert result["type"] == "form"
+    assert result["errors"] == {}
+
+    with patch(
+        "homeassistant.components.unifiled.config_flow.validate_input",
```

Comment:
We shouldn't patch the code we want to test. Instead patch `"homeassistant.components.unifiled.config_flow.unifiled"`.

Reply:
This is what was generated by the schaffold script. Though it was not catching an fault in the test script too. fixed in latest version